Python/Product/Django/TemplateParsing/TemplateArtifactCollection.cs

```diff
@@ -29,11 +29,14 @@ public TemplateArtifactCollection()
             LeftSeparator = RightSeparator = "";
         }
 
-        protected override ISensitiveFragmentSeparatorsInfo SeparatorInfo {
-            get {
-                return this;
-            }
-        }
+        // HACK: Does not build properly on Dev12

```

Comment:
Not really, except I was working on this in VS 2013 and so not being able to build was really annoying. This code is ifdef'd out for VS 2010 and VS 2012, so it's irrelevant.

AnalysisDictionary is not dead code - it's used for keeping track of variables. If anything, it's probably our biggest bottleneck (which is why changing it to a mostly thread-safe dictionary from a regular dictionary is such a good thing).